scripts/lib.sh

```diff
@@ -34,6 +34,32 @@ require() {
     command -v "$1" >/dev/null 2>&1
 }
 
+configure-sentry-cli() {
+    if [ -f "${venv_name}/bin/sentry-cli" ]; then
+        path="${venv_name}/bin/sentry-cli"
+    elif command -v "sentry-cli" >/dev/null 2>&1; then
+        path="sentry-cli"
+    elif [ -f "${venv_name}/bin/pip" ]; then
+        pip-install sentry-cli
+        path="${venv_name}/bin/sentry-cli"
+    else
+        curl -sL https://sentry.io/get-cli/ | SENTRY_CLI_VERSION="2.32.0" bash
```

Comment:
i don't think we should do this, it would be better to just simplify and prompt for `devenv sync` if `sentry-cli` isn't in the venv

Reply:
can also just remove all the following code for version checking, i think it's reasonable that devenv sync is run semi-regularly 

otherwise we're effectively invoking sentry-cli two times in do.sh